buildconfig/pygame-stubs/surface.pyi

```diff
@@ -1,49 +1,32 @@
-from typing import Any, List, Optional, Sequence, Text, Tuple, Union, overload, Iterable
+from typing import Any, Iterable, Optional, Sequence, Text, Tuple, Union, overload
+
 from pygame.bufferproxy import BufferProxy
 from pygame.color import Color
-from pygame.rect import Rect
-
 from pygame.math import Vector2
+from pygame.rect import Rect
 
-_ColorInput = Union[
-    Color, str, List[int], Tuple[int, int, int], Tuple[int, int, int, int]
-]
+_ColorInput = Union[Color, str, Sequence[int], Tuple[int, int, int], Tuple[int, int, int, int]]
 _RgbaOutput = Tuple[int, int, int, int]
 _RectStyle = Union[
     Rect,
     Tuple[float, float, float, float],
     Tuple[Tuple[float, float], Tuple[float, float]],
-    List[float],
-    List[Vector2],
+    Sequence[float],
+    Sequence[Vector2],
```

Comment:
I'm not sure this is the case at the moment.

```python
>>> pygame.Rect({1,2,3,4})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Argument must be rect style object
>>> pygame.Rect([1,2,3,4])
<rect(1, 2, 3, 4)>
```

Reply:
Will revert